dev-docs/source/Standards/InterfaceDocumentation.rst

```diff
@@ -12,6 +12,13 @@ Summary
 
 This page deals with the specifics of how to document Custom Interfaces. For a more general guide to the Mantid documentation system see :ref:`Documentation Guide For Devs <DocumentationGuideForDevs>`.
 
+Python Interface File Naming
+============================
+
+Python interface files, found in ``qt/python/mantidqtinterfaces/mantidqtinterfaces``, should be named with each word starting with a capital letter and underscores between the words.
+For example ``Engineering_Diffraction.py``. The corresponding documentation page, found somewhere within ``docs/source/interfaces``, should have the same name but with the underscores replaced with spaces (and a different file extension).
+For example ``diffraction/Engineering Diffraction.rst``. This should be followed so that documentation can be programmatically searched for, in order to check interfaces aren't added without the accompanying doc file.
```

Comment:
```suggestion
For example ``diffraction/Engineering Diffraction.rst``. This should be followed so that documentation can be programmatically searched for, in order to check interfaces aren't added without the accompanying documentation file.
```